packages/flask-aserto/pyproject.toml

```diff
@@ -32,15 +32,16 @@ python = "^3.8"
 Flask = {version = ">=2.0.0,<4.0.0", extras = ["async"]}
 Flask-Cors = ">=3.0.0,<5.0.0"
 grpcio = "^1.49.0"
-protobuf = "^4.21.0"
+protobuf = "^4.21.7"
```

Comment:
The `^` operator in poetry gives special treatment to zero versions.
`^4.21.7` is equivalent to `>=4.21.7,<4.22.0`
`^4.21.0` is equivalent to `>=4.21.0,<5.0.0`

I think that we want to tolerate minor version changes but not major versions. So I'd leave the version constraint as `^4.21.0`.